cloudwatch-event/run-task/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_iam_policy" "ecs_policy" {
+  policy = "${data.aws_iam_policy_document.ecs_policy.json}"
+}
+
+data "aws_iam_policy_document" "ecs_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["ECS:*"]
```

Comment:
if this is going to be ECS specific I would change the name of the folder & module to be `run-ecs-task`